client/src/components/DetailsPage/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import imgMeal from '../../assets/shrimppizza.jpg';
+import Button from '../utils/Button';
+import './style.css';
+
+const Details = () => {
```

Comment:
Make this a class component with state have the meal details in it:
```
state={
meal: {
id: 0,
amount: 0,
price: 0,
name: " ",
vegetables: [ ],
fruits: [ ],
nuts: [ ],
sauce: 0,
salt: 0,
spices: 0,
sugar: 0,
}, }
```
and then update this state depends on the radio buttons, checkboxes, inputs  values that the user chose. Send this meal by the props with the order component when the (show my order button) is clicked.

Have nice coding ^_^